user_docs/en/userGuide.t2t

```diff
@@ -220,6 +220,14 @@ If the command does not have a mode listed, it will work in any mode.
 To toggle touch modes, perform a 3-finger tap.
 %kc:endInclude
 
++++ Touch keyboard +++
+The touch keyboard is used to enter text and commands from a touchscreen.
+When focused on an edit field, you can bring up the touch keyboard by double-tapping the touch keyboard icon on the bottom of the screen.

```

Comment:
If the device is in Tablet mode, which may happen when no physical keyboard is connected to the machine (E.g. a Microsoft Surface Pro with its keyboard detached) the touch keyboard appears automatically when edit fields are focused.